ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp
+  steps:
+    pre:
+      - chain:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-provision
```

Comment:
you can directly use the provision/deprovision chain in `cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity`, not have to create a copy for the tp workflow.

Reply:
@jianlinliu  Appreciate it, is it correct now?

Reply:
right.